crates/virtio-queue-ser/Cargo.toml

```diff
@@ -13,5 +13,6 @@ edition = "2018"
 serde = { version = ">=1.0.27", features = ["derive"] }
 versionize = ">=0.1.6"
 versionize_derive = ">=0.1.3"
-virtio-queue = { path = "../../crates/virtio-queue" }
+# The `path` part gets stripped when publishing the crate.
+virtio-queue = { path = "../../crates/virtio-queue", version = ">=0.3.0" }
```

Comment:
This needs to be specified with `=` because it's the only way to map a version of virtio-queue to the corresponding virtio-queue-ser version.

Reply:
I've updated the dependency to `^0.3.0`, which should mean any version between `0.3.0` and `0.4.0` (but not including the latter). Does that work?